misk-jdbc/src/main/kotlin/misk/jdbc/Check.kt

```diff
@@ -0,0 +1,42 @@
+package misk.jdbc
+
+import java.util.EnumSet
+
+enum class Check {
+  FULL_SCATTER,
+  TABLE_SCAN,
+  COWRITE
+}
+
+object CheckDisabler {
```

Comment:
I liked this refactoring here.

Reply:
> This is a massive change in one of the misk libraries. How do you plan to test this before it gets rolled out to all misk services?

I tried running newswriter tests. They all pass except when merging is involved due to some weird classpath bug that I think is unrelated. So we should be all good!